trainer/openapi.json

```diff
@@ -38,7 +38,10 @@
                 "schema": {
                   "type": "object",
                   "properties": {
-                    "trainingBlobDownloadUrl": {
+                    "trainingDataUrl": {
```

Comment:
I don't think we need this file anymore, the client codegen reads the openapi spec from the dev server on localhost directly.

Reply:
Which file don't we need?

Reply:
I don't think we need `trainer/openapi.json`.